src/Plotly.NET/ChartAPI/Chart2D.fs

```diff
@@ -1547,7 +1547,67 @@ module Chart2D =
                 ?UpperName = UpperName,
                 ?UseDefaults = UseDefaults
             )
-
+            
+        /// <summary> Creates a Pareto chart. </summary>
+        /// <param name="keysValues">Sets the (key,value) pairs that are plotted as the size and key of each bar.</param>
+        /// <param name="Name">Sets the trace name. The trace name appear as the legend item and on hover</param>
+        /// <param name="Label">Sets the y axis label.</param>
+        [<Extension>]
+        static member Pareto
+            (
+                keysValues: seq<#IConvertible * float>
+                , [<Optional; DefaultParameterValue(null)>] ?Name: string
+                , [<Optional; DefaultParameterValue(null)>] ?Label: string
+            ) =
+            let orderedLabels, orderedValues =
+                keysValues
+                |> Seq.sortByDescending snd
+                |> Seq.unzip
+               
+            let sum = orderedValues |> Seq.sum
+            let topPaddingRatio = 0.05
+            let cumulativeSum = 
+                Seq.scan (+) 0. orderedValues
+                |> Seq.skip 1
+               
+            let paretoValues = 
+               Seq.zip orderedLabels cumulativeSum
+               |> Seq.map (fun (label,value) -> label, value / sum * 100.) 
+               
+            let bars = Chart.Column(Seq.zip orderedLabels orderedValues,?Name=Name)
+              
+            let points = 
```

Comment:
there is no need for 2 traces for points and lines here, you can show markers on a line chart with `ShowMarkers = true`.